runtime-modules/bounty/src/lib.rs

```diff
@@ -952,11 +1207,13 @@ decl_module! {
             Self::deposit_event(RawEvent::BountyFundingWithdrawal(bounty_id, funder));
 
             if Self::withdrawal_completed(&current_bounty_stage, &bounty_id) {
+                if Self::is_funding_expired(&bounty){
```

Comment:
I do not understand this condition, why is it required to return oracle reward?
In general I think this section having to do with cleanup requires more comments.